docs/source/tutorials/regression.ipynb

```diff
@@ -0,0 +1,972 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "ea0a577e",
+   "metadata": {},
+   "source": [
+    "# Find Label Errors in Regression Datasets"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d5e657d8",
+   "metadata": {},
+   "source": [
+    "In this 5-minute quickstart tutorial, we use cleanlab to find potential issues in regression datasets. Unlike classification algorithms, regressions algorithms can predict continuous quantities such as price, income, age and more. Here we consider a Student Grades dataset, which contains over 900 individuals who have three exam grades and some optional notes, each being assigned a final score. cleanlab automatically identifies examples in this dataset that have incorrect final scores.\n",
+    "\n",
+    "**Overview of what we’ll do in this tutorial:**\n",
+    "\n",
+    "- Train a simple Linear Regression model on the text embeddings to compute out-of-sample predictions.\n",
+    "- Use cleanlab's `CleanLearning.find_label_issues()` method to automatically compute out-of-sample predictions and identify potential label errors.\n",
+    "- Train a more robust version of the same model after dropping the identified label errors using CleanLearning."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "612a355a",
+   "metadata": {},
+   "source": [
+    "<div class=\"alert alert-info\">\n",
+    "Quickstart\n",
+    "<br/>\n",
+    "    \n",
+    "Already have an sklearn compatible `model`, regression data features `X` and given `y`? Run the code below to train your `model` and get label issues.\n",
+    "\n",
+    "\n",
+    "<div  class=markdown markdown=\"1\" style=\"background:white;margin:16px\">  \n",
+    "    \n",
+    "```python\n",
+    "\n",
+    "from cleanlab.regression.learn import CleanLearning\n",
+    "\n",
+    "cl = CleanLearning(model)\n",
+    "cl.fit(X, y)\n",
+    "label_issues = cl.get_label_issues()\n",
+    "preds = cl.predict(X_test) # predictions from a version of your model trained on auto-cleaned data\n",
+    "```\n",
+    "    \n",
+    "</div>\n",
+    "    \n",
+    "Is your model/data not compatible with `CleanLearning`? You can instead run cross-validation on your model to get out-of-sample `predictions`. Then run the code below to get label issue indices ranked by their inferred severity.\n",
+    "\n",
+    "\n",
+    "<div  class=markdown markdown=\"1\" style=\"background:white;margin:16px\">  \n",
+    "    \n",
+    "```python\n",
+    "\n",
+    "from cleanlab.regression.rank import get_label_quality_scores\n",
```

Comment:
[nit] can we move this import out of this section and down to the section below where this method is showcased?